graphql/resolvers.go

```diff
@@ -228,6 +228,14 @@ func (r *patchResolver) TaskCount(ctx context.Context, obj *restModel.APIPatch)
 	return &taskCount, nil
 }
 
+func (r *patchResolver) BaseVersionID(ctx context.Context, obj *restModel.APIPatch) (*string, error) {
+	baseVersion, err := model.VersionFindOne(model.VersionBaseVersionFromPatch(*obj.ProjectId, *obj.Githash))
```

Comment:
For this query, let's have it only return the ID since the version document can be very large. You can add a .Project after the return value of VersionBaseVersionFromPatch